content/participate/index.html.haml

```diff
@@ -0,0 +1,119 @@
+---
+layout: simplepage
+title: Participate and contribute
+section: participate
+---
+
+%p
+  There are numerous ways to engage with the Jenkins project and community.
+  Here are the most common ways to get you started.
+  Welcome aboard!
+
+
+%div
+  %div.col-lg-6
+    %h1
+      Participate
+
+    %p
+      As a participant, you take an active role in the community.
+
+    %h3
+      Meet
+    %p
+      Meet other Jenkins users and share your experiences.
+    %ul
+      %li
+        Organize or attend
+        %a{:href => '/projects/jam/'}
+          Jenkins Area Meetups
+      %li
+        Attend other events with a Jenkins presence, like SCALE or FOSDEM
+
+    %h3
+      Help
+    %p
+      Jenkins is a complex and extensible system, and many people are discovering it for the first time all the time. (TODO wordsmith)
+    %ul
+      %li
+        %a{:href => '/content/mailing-lists'}
+          jenkinsci-users mailing list
+      %li
+        %a{:href => '/content/chat'}
+          \#jenkins IRC channel on Freenode
+      %li
+        %a{:href => 'https://reddit.com/r/jenkinsci'}
+          \/r/jenkins on Reddit
+      %li
+        %a{:href => 'http://stackoverflow.com/tags/jenkins'}
+          Jenkins on Stack Overflow
+
+    %h3
+      Provide feedback
+    %p
+      We cannot fix what we don't know is broken. So let us know about it!
+    %ul
+      %li
+        %a{:href => 'https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue'}
+          How to report an issue in Jenkins
+      %li
+        The
+        %a{:href => '/changelog'}
+          changelog
+        has an integrated feedback mechanism. Just select
+        %em
+          Community ratings
+        and vote using the weather icons.
+      %li
+        Likewise, the
+        %a{:href => '/changelog'}
+          LTS changelog
+        provides the same functionality.
+
+  %div.col-lg-6
+    %h1
+      Contribute
+
+    %p
+      As a contributor, you actively help improve Jenkins and plugins by contributing code, documentation, translations, or tests.
+
+    %h3
+      Write code
+    %p
+      You enjoy writing code? There are numerous plugins and components for you to contribute to.
+      %a{:href => 'code'}
+        Learn more
+
+    %h3
+      Translate
+    %p
+      Jenkins is used all over the world, in numerous languages. If you speak languages other than English, you can
```

Comment:
I actually don't like either, numerous seems to weak to me and "all kinds of" makes me thing of different types of languages (programming, sign, spoken, etc).

I suggest a rephrase of:

> Jenkins is used all over the world by speakers of dozens of different languages, if you're fluent in languages other than English, consider improving support for those languages [a]by contributing to core and/or plugin localizations[/a]